### DISCUSSION TITLE
Having issues using Update-CrmOrganization from Microsoft.Crm.Powershell

### DISCUSSION THREAD

**Discussion by Indlad** [seanmcne/Microsoft.Xrm.Data.PowerShell#492] [Q&A] [answered]
Hi XRM powershell Team,
            I am trying to use Update-CrmOrganization to upgrade the Organization in Multi-Server environment and having issues successfully use this in Multi-Server environment. But works fine in single server environment.

I didn't see something in this Microsoft.Xrm.Data.Powershell module, are there any plans to have more methods to manage and Update organizations?

Regards,
Dan.

**Comment by seanmcne** [comment, marked as answer]
That's a different powershell module documented here - https://docs.microsoft.com/en-us/powershell/module/microsoft.crm.powershell/update-crmorganization?view=dynamics365ce-ps

This powershell module extends xrm toolings powershell connector (links should be in the readme on the main page and on our link in the powershell gallery). Hope that helps.
